c2corg_ui/views/document.py

```diff
@@ -431,6 +431,24 @@ def get_slug_for_title(title):
     return slug
 
 
+def is_collaborative(doc):
+    type = doc['type']
+
+    if type == 'r' or type == 'w' or type == 'a' or type == 'b':
+        return True
+
+    elif type == 'c':
+        if doc['article_type'] == 'collab':
+            return True
+
+    elif type == 'i':
+        if doc['image_type'] == 'collaborative':
+            return True
+
+    else:
```

Comment:
You don't need ``else`` here. Simply write ``return False`` as the last line of the function. If the conditions above are true this last line will not be executed.

Actually the code is incorrect with ``else``, because for images and articles, if the type check are not ``True`` no value is returned.

Please also remove all the extra and useless blank lines.